app/views/application/_subnavigation.html.erb

```diff
@@ -11,6 +11,7 @@
     <nav class='govuk-body govuk-!-margin-bottom-0 '>
       <% if current_user.super_admin? %>
         <%= link_to 'Organisations', admin_organisations_path, class: active_tab(admin_organisations_path) %>
+        <%= link_to 'Custom Organisations', admin_custom_organisations_path, class: active_tab(admin_custom_organisations_path) %>
```

Comment:
`Custom Organisations` feels OK here, definitely think we're implying that these screens will _create_ the organisation rather than just _allow_ people to create them, though.